src/Intersection.jl

```diff
@@ -475,8 +490,38 @@ end # _line_search
 end # quote
 end # load_optim
 
-# Symmetric case
-ρ(ℓ::AbstractVector{N}, cap::Intersection{N, S, <:LazySet};
-  algorithm::String="line_search", check_intersection::Bool=true,
-  kwargs...) where {N<:AbstractFloat, S<:Union{HalfSpace, Hyperplane}} = ρ(ℓ, cap.Y ∩ cap.X;
-  algorithm=algorithm, check_intersection=check_intersection, kwargs...)
+"""
+    _projection(ℓ, X, H::Union{Hyperplane{N}, Line{N}}; kwargs...) where {N}
+
+Given a compact and convex set ``X`` and a hyperplane ``H = \\{x: n ⋅ x = γ \\}``,
+calculate ...
+
+### Input
+
+- `ℓ`      -- direction
+- `X`      -- set
+- `H`      -- hyperplane
+
+### Output
+
+The support function of ``X ∩ H`` along direction ``ℓ``.
+"""
+function _projection(ℓ, X, H::Union{Hyperplane{N}, Line{N}};
```

Comment:
`ℓ` -> `d`?

Reply:
if u don't mind i would keep `ℓ` here so that it matches the original algorithm